### ISSUE TITLE
Urgent problem to fix to be able to order balsamic analyses

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/cgweb#235]
Original issue opened by @heronikdin in Scout: https://github.com/Clinical-Genomics/scout/issues/2175

Could somebody help?

Hi all!

I have talked with Kristina Lagerstedt and patrik grenfeldt to try and solve this issue, I am now hoping someone can help me since we are not getting any further.

For the TWIST myeloid samples we are trying to setup the same flow for ordering as the RD team. Meaning, we export a json file to orderportalen and then submit the order. However, when we choose "balsamic + analysis" instead of MIP, we get this message for samples ”please remove or save sample from case ”FXXXX”. Please note that there are no squares that are marked red, so in theory everything with the sample information is correct. And if we choose "MIP + analysis" the error message dissapears? So there seems to be something wrong with balsamic flow in orderportalen.

Please ask me if something is unclear, but this is extremely important to solve as we want to start sending the myeloid samples to clinical genomics very soon... we are already behind schedule. Gitub doesnt allow me to upload the json file, so I dont know how to share it. If needed I can email it!

Best Regards, HEro

**Comment by dnil** [comment]
Do you know what is up @patrikgrenfeldt @hassanfa?

**Comment by elevu** [comment]
@heronikdin replying here since I cannot tag you in the other issue, could you send me the json file at eleonora.maccacaro@scilifelab.se ?

**Comment by heronikdin** [comment]
> @heronikdin could you send me the json file at [eleonora.maccacaro@scilifelab.se](mailto:eleonora.maccacaro@scilifelab.se) ?

hi @elevu thank you so much for answering, I will send it to you now!

**Comment by elevu** [comment]
Received the file, I will get back to you asap!

**Comment by elevu** [comment]
@heronikdin during last week we fixed a bug with the Tumour validator in the Balsamic form flow not being displayed. **At least one sample per family needs to be a tumour.**

When I upload the json I got from you I get the error: 
![Screen Shot 2020-11-03 at 17 45 04](https://user-images.githubusercontent.com/6919697/98015064-b163bd80-1dfc-11eb-8fa2-17a62b6c1fbc.png)

This error was not visible last week, therefore I suspect this is the problem, sorry for the inconvenience! 

Let me know if you still don't manage to submit the samples!
